packages/plugins/test/mocha/cli/deploy.spec.ts

```diff
@@ -136,9 +132,19 @@ describe('Deploy CLI command', () => {
   })
 
   describe('With preview', () => {
-    const projectName = 'Simple_Project_1'
+    it('Executes deployment on local network twice', async () => {
```

Comment:
Updated this integration test to execute two deployments so that it serves the purpose of a regression test. The error would have occurred on the second deployment